collectors/apps.plugin/apps_groups.conf

```diff
@@ -298,13 +312,14 @@ solr: *solr.install.dir*
 airflow: *airflow*
 
 # -----------------------------------------------------------------------------
-# X
+# GUI
 
-X: X Xorg xinit lightdm xdm pulseaudio gkrellm xfwm4 xfdesktop xfce* Thunar
-X: xfsettingsd xfconfd gnome-* gdm gconf* dconf* xfconf* *gvfs gvfs* slim
-X: kdeinit* kdm plasmashell
-X: evolution-* firefox chromium opera vivaldi-bin epiphany WebKit*
-X: '*systemd --user*' chrome *chrome-sandbox* *google-chrome* *chromium* *firefox*
+gui: X Xorg xinit lightdm xdm gkrellm xfwm4 xfdesktop xfce* Thunar
+gui: xfsettingsd xfconfd gnome-* gdm gconf* dconf* xfconf* *gvfs gvfs* slim
+gui: kdeinit* kdm plasmashell
```

Comment:
When we start `kde` from terminal, we have these process:

```sh
bash-5.1$ ps aux| grep kde
thiago    1880  0.0  0.0   8636   344 tty1     S    11:14   0:00 /usr/lib64/libexec/kf5/start_kdeinit
thiago    1881  0.0  0.1 114220 29464 ?        Ss   11:14   0:00 kdeinit5: Running...
thiago    1894  0.0  0.5 1260840 85332 tty1    Sl   11:14   0:00 /usr/bin/kded5
thiago    1956  0.0  0.2 427664 47820 tty1     Sl   11:14   0:00 /usr/lib64/libexec/polkit-kde-authentication-agent-1
thiago    1961  0.0  0.3 495904 49464 tty1     Sl   11:14   0:00 /usr/lib64/libexec/org_kde_powerdevil
thiago    1980  0.0  0.3 360780 54832 tty1     Sl   11:14   0:00 /usr/lib64/libexec/kdeconnectd
thiago    2494  0.1  0.7 1403228 118872 ?      Sl   11:14   0:02 /usr/lib64/libexec/xdg-desktop-portal-kde
```

Should we also add them?

Reply:
the kdeinit stuff _should_ be getting matched already, and the process tree structure _should_ mean that the whole tree is getting matched.

Our existing kdeinit matching should probably be extended though.